root/work/merge.tt

```diff
@@ -62,8 +63,8 @@
                     <td>
                       [% work_attributes_list(entity) %]
                     </td>
-                  </tr>
```

Comment:
This seemed to be a completely unmatched `</tr>` (just TT things) but let me know if I'm being stupid.